@xen-orchestra/backups/docs/VM backups/README.md

```diff
@@ -76,12 +78,13 @@ job.start(data: { mode: Mode, reportWhen: ReportWhen })
 │  │  ├─ task.start(message: 'clean')
 │  │  │  ├─ task.warning(message: string)
 │  │  │  └─ task.end
-│  │  │
-│  │  │ // in case of delta backup
+│  │  └─ task.end
+|  ├─ task.start(message: 'clean-vm')
+│  │  ├─ task.warning(message: string)
```

Comment:
You can remove this line, warnings can happen for any tasks.

You can document specific warnings though, for instance those that you want to display in a specific way in the UI and/or backup report.